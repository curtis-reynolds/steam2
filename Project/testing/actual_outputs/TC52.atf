03_validgamenamee______sellstd_______100.00
00_sellstd_________SS_000200.00
