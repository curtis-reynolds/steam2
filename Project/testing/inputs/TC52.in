login
sellstd
sell
validgamenamee
100
sell
