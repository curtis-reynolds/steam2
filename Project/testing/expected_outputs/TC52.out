Login successful.
Game listed for sale.
A new game for sale has already been added in this session. You can only add one game per session.
